system-x/services/google/pom.xml

```diff
@@ -17,12 +17,27 @@
 
     <properties>
         <google-api.client.version>2.0.0</google-api.client.version>
-        <google-oauth-client-jetty.version>1.34.1</google-oauth-client-jetty.version>
         <google-auth-oauth.version>1.10.0</google-auth-oauth.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.google.api-client</groupId>
+                <artifactId>google-api-client</artifactId>
+                <version>${google-api.client.version}</version>
+            </dependency>
```

Comment:
is this needed somewhere?

Reply:
@tplevko both are transitive dependencies of both clients (sheets and storage) so this aligns them both to the same version

Reply:
okay, got it